source/agora/common/API.d

```diff
@@ -141,4 +141,24 @@ public interface API
     ***************************************************************************/
 
     public Block[] getBlocksFrom (ulong block_height, size_t max_blocks);
+
+
+    /***************************************************************************
+
+        Get the array of hashs the merkle branch.
```

Comment:
It's a bit hard to understand this documentation. Maybe to "get the array of hashes which form the merkle branch"? @Geod24 can you think of better wording?

Reply:
I assume "merkle branch" is actually a "merkle path", right ?

Reply:
In that case I suggest renaming the function to `getMerklePath`.